transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/store/ray_store.py

```diff
@@ -83,10 +121,7 @@ def get(self, key):
                 if data != None:
                     result = result + data
             except Exception as e:
-                print(
-                    "Error: %s. Ignoring the error",
-                    e,
-                )
+                print(f"Error: {e}. Ignoring the error")
```

Comment:
at 3 different places(line no. 92, 110 & 124) used the same string format. This might cause difficultly if we want to locate the failing point from the logs. can we have some uniqe keyword  in the string for each print statements.

Reply:
corrected.